### DISCUSSION TITLE
Disable recording? Alternatives?

### DISCUSSION THREAD

**Discussion by makalii-bouldercare** [mobile-dev-inc/Maestro#757] [Ideas]
Is it possible to disable the recording feature? Or to send it locally?
Also is it possible to more clearly convey that recordings will be sent to a different server?

**Comment by ArthurSav** [comment]
Hi, recordings are not enabled by default and are only produced when running `maestro record`. At them moment we don't support local only rendering but it is something we would like to add in the feature.

Note: All recording are deleted within 48 hours
